### PR TITLE
add  default font

### PR DESCRIPTION
**Pull request by moritzmock**
Related task: https://app.clickup.com/t/353nc82

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
+          'Arial',
+          'Noto Sans',
+          'sans-serif',
+          'Apple Color Emoji',
+          'Segoe UI Emoji',
+          'Segoe UI Symbol',
+          'Noto Color Emoji',
+        ],
+      },
+    },
```
Comment (mmalfertheiner) [inline tailwind.config.js:23]: I think you can remove this and instead do `@apply font-sans;` in the body. Could you please test that?

Comment (moritzmock) [reply tailwind.config.js]: @mmalfertheiner, yes, it is working as described! 

There is one change between the manual set fonts and the one imported from tailwind:
missing: OpenSans
new: ui-sans-serif

Should I add `OpenSans` again?

Comment (mmalfertheiner) [reply tailwind.config.js]: I would go with default Tailwind.

**Review by mmalfertheiner — APPROVED** [review]

**Comment by moritzmock** [comment]
Could you please merge it? It is blocked again for me...

File: tailwind.config.js [hunk 2]
```diff
@@ -1,26 +1,7 @@
...
-          'Noto Sans',
-          'sans-serif',
-          'Apple Color Emoji',
-          'Segoe UI Emoji',
-          'Segoe UI Symbol',
-          'Noto Color Emoji',
-        ],
-      },
-    },
+    extend: {},
```
Comment (mmalfertheiner) [inline tailwind.config.js:4]: extend is not required anymore, right?

Comment (mmalfertheiner) [reply tailwind.config.js:4]: No keep them, thats fine

Comment (moritzmock) [reply tailwind.config.js:4]: On the tailwind page it gives an example of a minimal config file https://v1.tailwindcss.com/docs/configuration
```// tailwind.config.js
module.exports = {
  future: {},
  purge: [],
  theme: {
    extend: {},
  },
  variants: {},
  plugins: [],
}```

**Comment by moritzmock** [comment]
@mmalfertheiner could you please merge it? If it looks OK too you!

**Pull request merged**